test/defaults.jl

```diff
@@ -109,34 +109,34 @@ end
     end
     @testset "FiniteDifferences" begin
         @testset "Derivative" begin
-            test_derivatives(fdm_backend1)
-            test_derivatives(fdm_backend2)
-            test_derivatives(fdm_backend3)
+            test_derivatives(fdm_backend1, test_types=false)
+            test_derivatives(fdm_backend2, test_types=false)
+            test_derivatives(fdm_backend3, test_types=false)
```

Comment:
Why skip these tests? Do we not expect them to pass?

Reply:
FDM seems to be behaving a bit oddly, often returning a 1 column matrix instead of a vector. Will revisit this later. I think the remaining issues are more urgent to fix now.

Reply:
there was only one test that caused this, I set `test_types` kwarg to true in most tests

Reply:
I opened an issue to track this https://github.com/JuliaDiff/AbstractDifferentiation.jl/issues/48.

Reply:
So I will mark this as resolved for now.